.github/workflows/scheduled.yaml

```diff
@@ -58,6 +58,14 @@ jobs:
       )
     name: revdepcheck ↩️
     uses: insightsengineering/r.pkg.template/.github/workflows/revdepcheck.yaml@main
+    with:
+      lookup-refs: |
```

Comment:
Do we need `deps-installation-method: setup-r-dependencies` in here?

Reply:
Looks like not, revdepcheck uses setup-r-dependencies already
https://github.com/insightsengineering/r.pkg.template/blob/main/.github/workflows/revdepcheck.yaml#L93